gradle.properties

```diff
@@ -20,7 +20,7 @@ kotlinCoroutinesVersion = 1.5.2
 kotlinxSerializationVersion = 1.3.0
 
 androidPluginVersion = 7.0.1
-classGraphVersion = 4.8.123
+classGraphVersion = 4.8.128
```

Comment:
lets upgrade to the latest `4.8.131` (https://github.com/classgraph/classgraph/releases)

Reply:
Sounds good. I changed it to upgrade to 4.8.131.